astropy/units/core.py

```diff
@@ -1540,18 +1525,14 @@ def score(compose):
 
     @lazyproperty
     def si(self) -> UnitBase:
-        """
-        Returns a copy of the current `Unit` instance in SI units.
-        """
+        """A copy of the unit in SI units."""
```

Comment:
Maybe "The unit expressed in terms of SI base units". I don't see why we have to say it is a copy...

Reply:
But the result is not always in base units:
```python
>>> from astropy import units as u
>>> u.Pa.si
Unit("N / m2")
```

Reply:
This property really should return the unit expressed in the SI base units. That would be predictable, simple to explain, simple to implement and consistent with the property's name. Changing the code is clearly beyond the scope of this pull request, but if we choose to do that then we will have to update the docstring too, which means that updating it here would not be very helpful. If you agree that we should change `.si` (and `.cgs`) then I will exclude them from this pull request.

Reply:
I'm not sure I agree it should be in base units, but certainly what it currently does makes no sense. See #17122 for a suggested alternative.

For the docstring, I'm fine with just something like "The unit expressed in SI units"

Reply:
#17122 looks reasonable to me, but I think it's that pull request that should update the `si` and `cgs` docstrings, so they should not be edited here.